portfolio.css

```diff
@@ -0,0 +1,89 @@
+
+body {
+  margin: 20px;
+  height: 100%;
+  width: 100%;
+  background-image: url('background.png');
+  background-size: 100% 100%;
+  background-repeat: no-repeat;
```

Comment:
Kindly make sure that your project is rendering correctly the mobile version. Your project should render correctly all the elements to screens from 375px to desktop size. Errors below 375px are not considerate. :+1: :smiley: :smiley: 

![image](https://user-images.githubusercontent.com/98363075/193268958-f2540ab8-45cb-4e53-ab93-0a2dfcf8173d.png)